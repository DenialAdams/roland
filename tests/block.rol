proc main() {
   {
      {
         {
            print("Hello world, I'm in a block!");
         }
      }
   }
}
