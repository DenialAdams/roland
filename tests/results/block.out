Hello world, I'm in a block!
